src/component/CmpCollapsible.tsx

```diff
@@ -62,6 +62,7 @@ export class CmpCollapsible extends Component<Props, State> {
                                 css={css`
                                     flex-grow: 1;
                                     display: flex;
+                                    min-height: 25px;
```

Comment:
Completely unrelated but it fixes a misaligned "arrow" icon on the collapsible buttons in the CMP modal, that was driving me crazy :upside_down_face:

Reply:
Love it when PR authors write these kind of comments, saves me time thinking about it ❤